addons/sourcemod/scripting/surftimer/timer.sp

```diff
@@ -176,18 +176,15 @@ public Action CKTimer2(Handle timer)
 				case 60:CPrintToChatAll("%t", "TimeleftSeconds", g_szChatPrefix, g_szMapName, timeleft);
 				case 30:CPrintToChatAll("%t", "TimeleftSeconds", g_szChatPrefix, g_szMapName, timeleft);
 				case 10:CPrintToChatAll("%t", "TimeleftSeconds", g_szChatPrefix, g_szMapName, timeleft);
-				case 3:CPrintToChatAll("%s ~~~ MAP ENDING ~~~", g_szChatPrefix);
-				case 2:CPrintToChatAll("%s ~~~ MAP ENDING ~~~", g_szChatPrefix);
-				case 1:CPrintToChatAll("%s ~~~ MAP ENDING ~~~", g_szChatPrefix);
-				case 0:CreateTimer(14.0, ForceNextMap, INVALID_HANDLE, TIMER_FLAG_NO_MAPCHANGE);	
-				case -1:
+				case 3:CPrintToChatAll("%s {green}3!", g_szChatPrefix);
```

Comment:
Maybe we could change this to Map ending in 3! and so on? Might be more obvious what is going on in the future! 😄

Reply:
I think it's personally cleaner this way, as you've got the multiple second warnings above. No one has ever second guessed what it is, if you want to add translations for it, go ahead. I just wanted the fix in to keep the dev branch stable.

Reply:
I think just keeping it as `Map restarting in 3` would be enough, no need to put in in the translations just yet, we can put a comment there about possibly adding it in the future to translations.

If you don't mind, I think it would be useful for those that don't know the code that good.

Reply:
> I think just keeping it as `Map restarting in 3` would be enough, no need to put in in the translations just yet, we can put a comment there about possibly adding it in the future to translations.
> 
> If you don't mind, I think it would be useful for those that don't know the code that good.

I disagree but yeah, too much text.